### ISSUE TITLE
Accessibility: Color contrast between states insufficient

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51828]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

The icon to show and hide the password goes from black to very dark gray when activated, with a color contrast ratio of 2.95:1 between the two states. This is a low priority issue because the actual visibility of the password also communicates the state of the button.

![T002_ShowHideIconDoNotRoleName](https://github.com/user-attachments/assets/bc1d77c0-0310-42a6-acdf-478c1a4c7bb7)

### Component or Page: 

Front-end Login Form on My Account Page

### Name affected element:

Show/Password icon

### Relevant code snippets:

`.woocommerce form .show-password-input.display-password::after, .woocommerce-page form .show-password-input.display-password::after {
    color: #585858;
}`

### Recommended fix:

Increase the color contrast between the states to have a color contrast ratio betwene the states of at least 3:1. The show/hide password button may be enhanced by also changing the visual appearance of the icon, such as adding and removing a line through the eye.

**Comment by darrenjacoby** [comment]
@vikrampm1 noting this should be solved with https://github.com/woocommerce/woocommerce/issues/51836. We can add extra QA steps there and close this one out.

**Comment by vikrampm1** [comment]
@darrenjacoby Can we link #53060 to this issue along with adding extra QA steps there? That way this will also get closed out with #51836 when the PR gets merged.

**Comment by darrenjacoby** [comment]
@vikrampm1 done!
